docker/Dockerfile

```diff
@@ -77,6 +77,11 @@ ENV TORCH_CUDA_ARCH_LIST=${torch_cuda_arch_list}
 # Override the arch list for flash-attn to reduce the binary size
 ARG vllm_fa_cmake_gpu_arches='80-real;90-real'
 ENV VLLM_FA_CMAKE_GPU_ARCHES=${vllm_fa_cmake_gpu_arches}
+
+# TESTING: install xformers from source to test 2.7.0 final RC
+RUN --mount=type=cache,target=/root/.cache/uv \
+    TORCH_CUDA_ARCH_LIST='7.5 8.0 8.6 8.9 9.0+PTX' \
```

Comment:
for blackwell support we will wnat to add sm100 here, although the +PTX should handle this..

Reply:
Yes 10.0 as well as 12.0.  +PTX is not enough.  (Same comment in the several places this sequence appears.)  cc @kushanam

Reply:
As the size of this PR is relative big already and its signals are ready, let me add this change to a subsequent PR. It would be clearer that way I guess